README.md

```diff
@@ -26,5 +26,16 @@ Repository for TDR transfer code
 
 * Start the application using `sbt run`
 
+* If css is not displaying on localhost:9000/dashboard run `npm run build`
+
+* If npm is not installed install [nvm](https://github.com/nvm-sh/nvm) in root directory.
+
+* Once nvm is installed:
+    `nvm install 13.6`
+
+* `cd` into tdr-transfer-frontend in terminal
+
+* run  `npm install` again, then `npm run build`. CSS should now appear on localhost.
```

Comment:
Just noticed that this README didn't actually say how to run the application.

Is it just `sbt run` at the moment, or do you need to set any environment variables? Either way, could you add another subsection to explain how to start Play?

Reply:
No environmental variables needed. Instructions on how to run have now been moved to the correct place as well